terratest/test/basic_test.go

```diff
@@ -110,6 +110,6 @@ func TestBasicExample(t *testing.T) {
 	t.Run("Type AAAA returns Rcode 3", func(t *testing.T) {
 		msg, err := DigMsg(t, "localhost", 1053, "host1.example.org", dns.TypeAAAA)
 		require.NoError(t, err)
-		assert.Equal(t, dns.RcodeNameError, msg.Rcode)
+		assert.Equal(t, dns.RcodeSuccess, msg.Rcode)
```

Comment:
does `dns.RcodeSuccess` mean `NOERROR`  and `NODATA (empty reply)`

can you 
 - assert both values ?
 - add the assertion for the rest of related tests ?

Reply:
What happen if we make corrupted AAAA request ? is `dns.RcodeSuccess` expected result ?

Reply:
@kuritka thanks I added NODATA test